scripts/globals/bcnm.lua

```diff
@@ -275,7 +275,7 @@ local battlefields =
     --  { 10,   74, 1551 },   -- Crustacean Conundrum (BS20) -- TODO: You can only do 0-2 damage no matter what your attack is
         { 11,   75, 1552 },   -- Grove Guardians (BS30)
     --  { 12,   76, 1553 },   -- The Hills are Alive (KS99) -- TODO: Tartaruga Gigante is not coded
-    --  { 13,   77, 1131 },   -- Royal Jelly (BS40) -- TODO: all combat mechanics, loot
+        { 13,   77, 1131 },   -- Royal Jelly (BS40) -- TODO: all combat mechanics, loot
```

Comment:
looks like you added the combat mechanic and loot, maybe that TODO needs to be removed?

also awesome  for contributing with this BCNM 😄  its one of my favourites 😄

Reply:
done